### PR TITLE
New CME Cert Layout

### PR DESCRIPTION
**Pull request by caesar2164**
- Left aligned
- Date at top right
- Student Name and Course Title automatically shrinks according to length

@stvstnfrd, review please.

### CODE REVIEW

**Comment by stvstnfrd** [comment]
BTW: Travis tests should now only be running once. Prior, they were configured to run on PR changes _and_ any pushes, which meant PRs would see the tests run twice.

File: cert-data.yml [hunk 1]
```diff
@@ -12,6 +12,13 @@ edX/DemoX_v3/Demo_Course_v3:
     Pass: 'This person passed.'
     Distinction: 'This person is a complete overachiever!'
 
+Test/CME/Renderer:
+  LONG_ORG: (╯°□°）╯︵ ┻━┻
+  LONG_COURSE: (╯°□°）╯︵ ┻━┻
```
Comment (stvstnfrd) [inline cert-data.yml:16]: Maybe we want better unicode names?

File: openedx_certificates/renderers/cme.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+"""
+TODO: put docstring here
```
Comment (stvstnfrd) [inline openedx_certificates/renderers/cme.py]: I forgot this.

File: cert-data.yml (hunk 1 above)
Comment (caesar2164) [inline cert-data.yml:16]: Also, `LONG_ORG` is unnecessary in CME and v3_dynamic courses...

**Pull request merged**